test/test_autograd.py

```diff
@@ -2797,6 +2797,49 @@ def run_test(upper, dims):
         for upper, dims in product([True, False], [(3, 3), (5, 3, 3), (4, 3, 2, 2)]):
             run_test(upper, dims)
 
+    @skipIfNoLapack
+    def test_symeig_degenerate(self):
+        # test symeig backward if there are degenerate/repeated eigenvalues
+        dtype = torch.double
```

Comment:
Could you please add an explanation what `if there are degenerate/repeated eigenvalues` really mean for completeness?

Reply:
I slightly rephrased it to `test symeig backward in cases where there are degenerate/repeated eigenvalues`, hopefully it is clearer.

Reply:
What I meant is to explain in the test how decide that the problem is ill-conditioned.